app/src/main/java/com/quran/labs/androidquran/common/QariItem.kt

```diff
@@ -0,0 +1,21 @@
+package com.quran.labs.androidquran.common
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+class QariItem(
+  val id: Int,
+  val name: String,
+  val url: String,
+  val path: String,
+  var databaseName: String? = null
```

Comment:
will merge this, but please change this to be a `val` - the code in `init` doesn't add much value in this case.